src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITHM = "SHA-256";
+
+    public HashUtil() {
+    }
+
+    public static String getHash(String pas, byte[] salt) {
```

Comment:
```suggestion
    public static String getHash(String password, byte[] salt) {
``` 
cmon, don't be lazy :)